PhysicsTools/PatAlgos/plugins/PATMuonSlimmer.cc

```diff
@@ -56,6 +61,8 @@ pat::PATMuonSlimmer::PATMuonSlimmer(const edm::ParameterSet & iConfig) :
         for (const edm::InputTag &tag : pf2pc) pf2pc_.push_back(consumes<edm::Association<pat::PackedCandidateCollection>>(tag));
     }
 
+    pc_ = consumes<pat::PackedCandidateCollection>(iConfig.getParameter<std::vector<edm::InputTag>>("packedPFCandidates")[0]);
```

Comment:
is pc_ supposed to be only one collection?
@gpetruc this vector input of  packedPFCandidates was added for bad muon tagging.
Please check and clarify if pc_ is only one collection .. and that it also has to be the first on the list.